app/code/Magento/ConfigurableProduct/Pricing/Render/FinalPriceBox.php

```diff
@@ -45,7 +59,8 @@ public function __construct(
         array $data = [],
         LowestPriceOptionsProviderInterface $lowestPriceOptionsProvider = null,
         SalableResolverInterface $salableResolver = null,
-        MinimalPriceCalculatorInterface $minimalPriceCalculator = null
+        MinimalPriceCalculatorInterface $minPriceCalculator = null,
```

Comment:
constructor argument names must not be changed. This could break installations where an implementation is injected directly by argument name

Reply:
fixed